Modules/TPC/src/PID.cxx

```diff
@@ -58,6 +60,8 @@ void PID::initialize(o2::framework::InitContext& /*ctx*/)
   mQCPID.setPIDCuts(cutMinNCluster, cutAbsTgl, cutMindEdxTot, cutMaxdEdxTot, cutMinpTPC, cutMaxpTPC, cutMinpTPCMIPs, cutMaxpTPCMIPs, runAsyncAndTurnOffSomeHistos);
   mQCPID.setCreateCanvas(createCanvas);
   mQCPID.initializeHistograms();
+  pSeparationPower = new TProfile("pSeparationPower", "pSeparationPower", nPars, 0., (float)nPars);
+  getObjectsManager()->startPublishing(pSeparationPower);
```

Comment:
This is still a memory leak, since there is no `delete` for this `new`. So either make the member a unique_ptr
```
std::unique_ptr<TProfile> mSeparationPower;
```
and here
```
mSeparationPower.reset(new TProfile...);
getObjectsManager()->startPublishing(mSeparationPower.get());
```
or put it on the stack, so the data member would be:
```
TProfile mSeparationPower("mSeparationPower", "mSeparationPower", nPars, 0., (float)nPars);
```
and
```
 getObjectsManager()->startPublishing(&mSeparationPower);
```
and change the access operator below accordingly from `->` to `.`